### PR TITLE
Add item database

### PR DESCRIPTION
**Pull request by RichAWarren**
added add_item_database backend helper

### CODE REVIEW

**Comment by matthewglover** [comment]
Can you check the tests?

**Comment by RichAWarren** [comment]
Tests now passing

File: server/lib/add_item_database.js [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+const { getGeoLocation } = require('./geolocation');
+
+function addItemDatabase(connection, item, callback) {
+  const title = item.title;
+  const description = item.description;
+  const postcode = item.postcode;
+  const category = item.category;
+  const imageUrl = item.image_url;
+  const userId = item.user_id;
+  const client = pgClient(connection);
```
Comment (matthewglover) [inline server/lib/add_item_database.js]: change to work with updated client connection

File: server/lib/get_item_database.js [hunk 2]
```diff
@@ -0,0 +1,51 @@
+const pgClient = require('./pg_client');
+
+
+const getItemDatabase = (connection, keywordInput,
+  categoryInput, userIdInput, itemIdInput, callback) => {
+  const client = pgClient(connection);
```
Comment (matthewglover) [inline server/lib/get_item_database.js]: update to new pg

File: server/lib/get_item_database.js [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+    user_id: userIdInput,
+    id: itemIdInput,
+  };
+  // if (obj.title !== null) {
+  //   obj.title = `%${keywordInput}%`;
+  // }
+  const array = [];
+  let partquerystring = '';
+  let count = 1;
+  Object.keys(obj).forEach((key) => {
```
Comment (matthewglover) [inline server/lib/get_item_database.js]: use Object.assign

File: server/lib/update_item_database.js [hunk 4]
```diff
@@ -0,0 +1,46 @@
+// import { getUserDatabase } from './get_user_database.js';
+
+const pgClient = require('./pg_client');
+const getItemDatabase = require('./get_user_database').getUserDatabase;
+
+const updateItemDatabase = (connection, userId, updates, callback) => {
```
Comment (matthewglover) [inline server/lib/update_item_database.js]: use updated pg module

File: server/lib/update_item_database.js [hunk 5]
```diff
@@ -0,0 +1,46 @@
+// import { getUserDatabase } from './get_user_database.js';
```
Comment (matthewglover) [inline server/lib/update_item_database.js]: Delete file

File: test_server/lib/testdb.js [hunk 6]
```diff
@@ -3,6 +3,11 @@ import test from 'ava';
 import pg from 'pg';
 import { exec } from 'child_process';
 import path from 'path';
+import { addUserDatabase } from '../../server/lib/add_user_database.js';
+import { getUserDatabase } from '../../server/lib/get_user_database.js';
+import { updateUserDatabase } from '../../server/lib/update_user_database.js';
+import { addItemDatabase } from '../../server/lib/add_item_database.js';
+import { getItemDatabase } from '../../server/lib/get_item_database.js';
 
 const dumpDbFile = path.join(__dirname, '..', 'fixtures', 'dumpdb.sql');
```
Comment (matthewglover) [inline test_server/lib/testdb.js]: update database files

**Comment by mcelearr** [comment]
Closes #17

**Pull request merged**